src/main/java/DukeWriteFile.java

```diff
@@ -0,0 +1,44 @@
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Writes the current tasks in the lst to a text document
+ * whenever the task list changes.
+ */
+public class DukeWriteFile {
+
+	protected static String filePath;
+
+	public DukeWriteFile(String filePath) {
+
+		this.filePath = filePath;
+	}
+
+
+	/**
+	 * Writes the current task in the list to the text document.
+	 *
+	 * @param textToaAdd Current tasks in the list.
+	 * @throws IOException
+	 */
+	public static void writeToFile(String textToaAdd) throws IOException {
+		FileWriter fw = new FileWriter(filePath);
+		fw.write(textToaAdd);
+		fw.close();
+	}
+
+
+	/**
+	 * Appends new tasks to the text document instead of overwriting
+	 * the current content.
+	 *
+	 * @param textToAppend New task to be appended to the text document.
+	 * @throws IOException
+	 */
+	public static void appendToFile(String textToAppend) throws IOException {
+		FileWriter fw = new FileWriter(filePath, true);
+		fw.write(textToAppend);
+		fw.close();
+	}
+
+}
```

Comment:
It'd be better if you moved the writing logic into this class from `Duke`. At the moment this class just has utility functions for writing to files...